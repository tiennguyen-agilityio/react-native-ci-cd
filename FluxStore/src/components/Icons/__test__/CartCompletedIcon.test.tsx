import {render} from '@testing-library/react-native';

import {CartCompletedIcon} from '../CartCompletedIcon';

describe('CartCompletedIcon', () => {
  it('renders correctly (matches snapshot)', () => {
    const {toJSON} = render(<CartCompletedIcon />);
    expect(toJSON()).toMatchSnapshot();
  });

  it('renders correctly (matches snapshot) with isActive true', () => {
    const {toJSON} = render(<CartCompletedIcon isActive />);
    expect(toJSON()).toMatchSnapshot();
  });
});
