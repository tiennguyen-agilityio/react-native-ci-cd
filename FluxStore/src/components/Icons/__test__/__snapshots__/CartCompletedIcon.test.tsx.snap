// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartCompletedIcon renders correctly (matches snapshot) 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight={110}
  bbWidth={110}
  fill="none"
  focusable={false}
  height={110}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 110,
        "width": 110,
      },
    ]
  }
  vbHeight={110}
  vbWidth={110}
  width={110}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M76.456 51.777c-.63 0-1.255.023-1.876.062l-1.561-25.95H59.982v-6.626C59.983 8.641 51.342 0 40.72 0 30.097 0 21.456 8.641 21.456 19.263v6.626H8.407L4.433 97.056h47.826C57.487 104.855 66.382 110 76.456 110c16.052 0 29.111-13.06 29.111-29.111 0-16.052-13.059-29.112-29.111-29.112zM27.901 19.263c0-7.067 5.75-12.818 12.818-12.818s12.818 5.75 12.818 12.818v6.626H27.901v-6.626zm9.722 26.66l3.25 2.158V58.25h-6.5V48.08l3.25-2.157zM11.248 90.61l3.254-58.276h6.954v9.722h6.445v-9.722h6.5v7.993l-6.473 4.297v20.07h19.39v-20.07l-6.472-4.297v-7.993h12.691v9.722h6.446v-9.722h6.966l1.242 20.64C56.152 56.545 47.345 67.707 47.345 80.89c0 3.407.59 6.68 1.671 9.721H11.248zm65.208 12.945c-12.498 0-22.666-10.168-22.666-22.666 0-12.498 10.168-22.666 22.666-22.666 12.498 0 22.666 10.168 22.666 22.666 0 12.498-10.168 22.666-22.666 22.666z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
    <RNSVGPath
      d="M73.966 82.11l-6.953-6.953-4.557 4.557 11.51 11.51 16.49-16.491-4.557-4.558L73.966 82.11z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`CartCompletedIcon renders correctly (matches snapshot) with isActive true 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight={110}
  bbWidth={110}
  fill="none"
  focusable={false}
  height={110}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 110,
        "width": 110,
      },
    ]
  }
  vbHeight={110}
  vbWidth={110}
  width={110}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M76.456 51.777c-.63 0-1.255.023-1.876.062l-1.561-25.95H59.982v-6.626C59.983 8.641 51.342 0 40.72 0 30.097 0 21.456 8.641 21.456 19.263v6.626H8.407L4.433 97.056h47.826C57.487 104.855 66.382 110 76.456 110c16.052 0 29.111-13.06 29.111-29.111 0-16.052-13.059-29.112-29.111-29.112zM27.901 19.263c0-7.067 5.75-12.818 12.818-12.818s12.818 5.75 12.818 12.818v6.626H27.901v-6.626zm9.722 26.66l3.25 2.158V58.25h-6.5V48.08l3.25-2.157zM11.248 90.61l3.254-58.276h6.954v9.722h6.445v-9.722h6.5v7.993l-6.473 4.297v20.07h19.39v-20.07l-6.472-4.297v-7.993h12.691v9.722h6.446v-9.722h6.966l1.242 20.64C56.152 56.545 47.345 67.707 47.345 80.89c0 3.407.59 6.68 1.671 9.721H11.248zm65.208 12.945c-12.498 0-22.666-10.168-22.666-22.666 0-12.498 10.168-22.666 22.666-22.666 12.498 0 22.666 10.168 22.666 22.666 0 12.498-10.168 22.666-22.666 22.666z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
    <RNSVGPath
      d="M73.966 82.11l-6.953-6.953-4.557 4.557 11.51 11.51 16.49-16.491-4.557-4.558L73.966 82.11z"
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
